### DISCUSSION TITLE
Kudos on a great platform for sand tables

### DISCUSSION THREAD

**Discussion by CharlieFoxUK** [texx00/sandypi#89] [General]
Hi, I recently finished a grbl based x-y sand table and jumped straight into using octoprint with some plugins.  I quickly shifted over to your application as it looked way more professional and suited to purpose.  I'm very glad I did as the octoprint had way too many little issues and a severe lack of features compared to yours.  

I found the install process still a little daunting simply because I'd never used docker before so it took me a while to figure out installing docker on the pi.  For some reason I couldn't get docker compose to install with trying various instructions on different websites and ended up simply trying to install it within a python script then voila!   It seems like they are now packaging docker and docker compose into docker desktop and so maybe this has something to do with it.

Anyway thanks for the effort put into this it has allowed all my hard work on my table feel well worth it.

**Comment by texx00** [comment]
Thanks for the appreciation and the feedback!

Trust me if I tell you the installation is easier through docker... It used to be "manual" and it was every time a mess... 
This is the second time in recent time I hear somebody complaining about the docker and docker-compose installation. I'm going to open an issue in order to remember about this and maybe create a dedicated guide. I was working on it about one year ago and it was working flawlessly but as you said they changed something with their releases.

The project is still bare bones. I would like to have the time to work on it, fixing bugs and adding features but at the moment I'm very busy and I cannot see when I will have the time to go back on this. Hopefully soon ;)
